test/functional/src/test_output_buffer01.c

```diff
@@ -0,0 +1,37 @@
+/*****************************************************************************
```

Comment:
I would be worth putting a similar example in the `examples` directory. I can update the docs after that since the steps are probably not obvious.